pkg/sql/catalog/systemschema/system.go

```diff
@@ -796,6 +796,14 @@ CREATE TABLE system.tenant_usage (
   -- Cumulative usage statistics, encoded as roachpb.TenantConsumption.
   total_consumption BYTES,
 
+  -- Computed rates of consumption for metrics used in tenant modeling, encoded
+  -- as roachpb.TenantConsumptionRates.
+  current_rates BYTES,
```

Comment:
Why is current_rates and next_rates a protobuf instead of a double? If this were a double, it would be easier to inspect the table when troubleshooting.

Reply:
Because I want to add at least one more rate into the struct, for estimated CPU. I need to know the number of estimated CPUs in use in order to "smear" background usage. This is an upcoming PR.